docs/data-modeling/json-unfolding.md

```diff
@@ -68,3 +68,4 @@ For example, if you upload a CSV with JSON in it, you might need to update the d
 - [PostgreSQL](../databases/connections/postgresql.md)
 - [MySQL](../databases/connections/mysql.md)
 - [Druid (JDBC)](../databases/connections/druid.md)
+- [BigQuery](../databases/connections/bigquery.md) (always enabled)
```

Comment:
In other DBs you have to enable it, but in BQ it's on by default. I'd still like to call out explicitly that it's there in BQ